src/test/java/ch/wisv/areafiftylan/UserRestIntegrationTest.java

```diff
@@ -249,23 +249,7 @@ public void testCreateUserIsDisabled() {
     }
 
     @Test
-    public void createUserNoEmail() {
```

Comment:
Because we already have a [`createUserMissingEmailField()`](https://github.com/AreaFiftyLAN/lancie-api/pull/406/files#diff-1a637b6c2fc39b5c44162281c7a849c2R252) test. It was left over from the move from `username` + `email` to only `email` as username.